engine/pom.xml

```diff
@@ -103,11 +103,11 @@
     </dependency>
 
     <dependency>
-      <groupId>javax.persistence</groupId>
-      <artifactId>persistence-api</artifactId>
+      <groupId>org.hibernate.javax.persistence</groupId>
+      <artifactId>hibernate-jpa-2.0-api</artifactId>
```

Comment:
Should we document this change in the migration guide, or it would not impact users at all?

Reply:
I don't think this needs a mention. JPA 2.0 is part of Java EE 6 and Java EE 6 was e.g. already supported by JBoss 7.2.